napari/_qt/qt_main_window.py

```diff
@@ -157,7 +158,26 @@ def _add_file_menu(self):
         )
         open_folder.triggered.connect(self.qt_viewer._open_folder)
 
-        screenshot = QAction('Screenshot', self._qt_window)
+        save_all_layers = QAction('Save all layer(s)...', self._qt_window)
+        # Save ctrl+S for saving the entire viewer state?
+        save_all_layers.setShortcut('Ctrl+Shift+S')
+        save_all_layers.setStatusTip('Save all layers')
+        save_all_layers.triggered.connect(
+            lambda: self.qt_viewer._save_layers(selected=False)
+        )
+
+        save_selected_layers = QAction(
+            'Save selected layer(s)...', self._qt_window
+        )
+        # Save ctrl+S for saving the entire viewer state?
```

Comment:
As I recall we decided Ctrl-S for save selected layers, Ctrl-Shift-S for save all layers, and Ctrl-Alt-Shift-S for saving the viewer state.

Reply:
If we're making `ctrl-S` the simpler choice for save-selected, should we also make `selected=True` the default value for `viewer.layers.save`? Under the same assumption of that is what people will want to do more frequently?

Reply:
this i like less. I liked that `layers.save()` behaved as a composite variant on `layer.save()`, it also sets us up for using layer groups as temporary disposable collection of layers.

Reply:
makes sense